dashboard/config/scripts_json/vpl-csd-2022.script_json

```diff
@@ -8,20 +8,6 @@
       "is_course": true,
       "is_migrated": true,
       "student_detail_progress_view": true,
-      "teacher_resources": [
```

Comment:
These must've been added recently. Is anything going to break for these PL scripts?